src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -97,9 +98,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        Gender updatedGender = editPersonDescriptor.getGender().orElse(personToEdit.getGender());
```

Comment:
Would you consider updating the `MESSAGE_USAGE` in this class as well?

Reply:
Done